docs/support/tutorials/gis/eo_guide.md

```diff
@@ -0,0 +1,259 @@
+
+[**Google Cloud**](https://cloud.google.com) provides some [public EO datasets in Google Cloud Storage](https://cloud.google.com/storage/docs/public-datasets), inlcuding Sentinel-2 L1C and Landsat Collection 1 data. Google Cloud also offers the possibility for virtual machines, with both CPU and GPU.
+ and <a href="https://cloud.google.com/storage/docs/public-datasets/landsat" aria-label="Landsat: Collection 1 data on Google cloud storage">Landsat: Collection 1</a>. Data can be downloaded from here for example with [FORCE](../../../apps/force.md).
+
+[**Terramonitor**](https://www.terramonitor.com/services/analysis-ready) provides pre-prosessed, analysis ready Sentinel-2 data from Finland available between 2018-2020.[Contact Terramonitor about pricing](https://www.terramonitor.com/services/analysis-ready#contact)
+
+[**Sentinelhub**](https://www.sentinel-hub.com/explore/) provides among others worldwide <a href="https://collections.sentinel-hub.com/sentinel-2-l2a/" aria-label="Sentinel 2 on sentinelhub">Sentinel 2</a> and <a href="https://collections.sentinel-hub.com/sentinel-1-grd/" aria-label="Sentinel 1 on sentinelhub">Sentinel 1</a> data. Requires [sentinelhub subscription](https://www.sentinel-hub.com/pricing/).
+
+!!! default "STAC"
+
+    Many data providers and companies also provide a Spatio Temporal Asset Catalog (STAC) of their and other datasets. These catalogues help in finding available data based on time and location with the possibility for multiple additional filters, such as cloud cover and resolution. The [STAC Index](https://www.stacindex.org/) provides a nice overview of available catalogues from all over the world. The STAC Index page also includes many resources for learning and utilizing STAC. Check out also CSC's [examples for utilizing STAC from Python](https://github.com/csc-training/geocomputing/blob/master/python/STAC).
+
+## Where can I store the data?
+
+What to consider when chosing a storage location:
+
+* Raw vs intermediate vs final result data
+    * What needs to be stored?
+    * Storage space?
+* Accessibility
+    * Sensitive data?
+    * Who needs to have access?
+    * How needs the data to be accessed?
+    * Intended usage?
+* Maintenance needs?
+* Metadata needs?
+
+In general, data can be stored on the [supercomputer disk](../../../computing/disk.md) or in the object storage Allas ([CSC Allas overview](https://research.csc.fi/-/allas), [CSC Allas guide](../../../data/Allas/index.md)). On the supercomputer, data can be stored on ```/scratch/project_xxx``` with xxx being your project number. Smaller amounts can also be stored short term on the computing nodes ```$LOCAL_SCRATCH``` during processing. In Allas, data is stored in so-called buckets, and can be accessed or transferred as part of the computing job, see also [CSC's webinar on Allas for spatial data](https://www.youtube.com/watch?v=mnFXe2-dJ_g).
```

Comment:
```suggestion
### Storage in CSC computing environment

* [Local storage in HPC:](../../../computing/disk.md) 
	* The default data storage is in ```/scratch/project_xxx``` with xxx being your project number. 
	* Smaller amounts can also be stored short term on the computing nodes ```$LOCAL_SCRATCH``` during processing. 

* (Object storage Allas)[CSC Allas guide](../../../data/Allas/index.md)). Data is stored in buckets, and can be accessed or transferred as part of the computing job. Allas can be accessed from all o See also [CSC's webinar on Allas for spatial data](https://www.youtube.com/watch?v=mnFXe2-dJ_g).
```

Reply:
Hmm,  I wonder if we at some point should call say that now it is only about CSC services? Now the HPC disk details come rather suddenly without even intro to HPC..